### ISSUE TITLE
Account Registration

### ISSUE THREAD

**Issue by ainsfletcher** [ainsfletcher/CSCI3308-Spring2023-Match-Play-016-Group1#2]
User should be able to register for an account by creating a username, password, and inputting their information.

**Comment by TurnerTastic1** [comment]
> User should be able to register for an account by creating a username, password, and inputting their information.

Yes they should. I'm a good golfer, me and Tiger Woods are ready to join!
